source/Core/BSP/Sequre/configuration.h

```diff
@@ -144,7 +144,8 @@
 #define POWER_LIMIT_STEPS  5
 #define OP_AMP_GAIN_STAGE  536
 #define TEMP_uV_LOOKUP_S60
-#define USB_PD_VMAX 12 // Maximum voltage for PD to negotiate
+#define USB_PD_VMAX        12  // Maximum voltage for PD to negotiate
+#define USB_PD_TIMEOUT     1   // Default Timeout for USB-PD Protocol negotiation in x100ms
```

Comment:
I think it would be wiser for us to raise this to `5` while we are here

Reply:
5 is causing problems if I try to power it on my Macbook Pro M2. I'll make this choice up to you.

Reply:
If it causes issues you can leave it at `1` then. I don't have any apple devices to test with so 🤷🏼 .